devops/scripts/register_bundle_with_api.sh

```diff
@@ -97,19 +105,24 @@ if [[ -z ${acr_name:-} ]]; then
     usage
 fi
 
-if [[ -z ${BUNDLE_TYPE:-} ]]; then
+if [[ -z ${bundle_type:-} ]]; then
```

Comment:
Why the casting change? This is an environment variable ad I think it supposed to be upper case per [Bash best practices](https://google.github.io/styleguide/shellguide.html#s7-naming-conventions).
(anyway, you left a bunch with their upper case casting)

Reply:
my bad. Hard to do this over the phone...